streaming/index.js

```diff
@@ -35,6 +35,29 @@ dotenv.config({
 
 initializeLogLevel(process.env, environment);
 
+
+/**
+ * @typedef Filter
+ * @property {number} id
+ * @property {string} title
+ * @property {string[]} context
+ * @property {Date} expires_at
+ * @property {'warn' | 'hide'} filter_action
+ */
+
+/**
+ * @typedef KeywordFilter
+ * @property {Array<[string, boolean]>} keywords
```

Comment:
tbh, I'm not really a fan of this shape. I think we'd have clearer code in a different shape, though I know why we have this shape.